lib/ddtrace/contrib/rails/framework.rb

```diff
@@ -47,6 +48,10 @@ def self.configure(config)
             port: datadog_config[:trace_agent_port]
           )
 
+          # set default tracer tags
+          datadog_config[:tracer].set_tags('env' => datadog_config[:env])
+          datadog_config[:tracer].set_tags(datadog_config[:tags])
```

Comment:
Wondering about the order here, as I understand it, what is in `env:` would be overridden by the contents of `tags:`? I, intuitively, would think it the other way around, but won't enforce it, just curious to know why you chose that order.